flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/StreamTableEnvironmentTest.scala

```diff
@@ -322,6 +322,10 @@ class StreamTableEnvironmentTest extends TableTestBase {
     } catch {
       case e: TableException =>
         assertTrue(e.getMessage.contains("Only default behavior is supported now"))
```

Comment:
Remove this. It seems no one will getinto this branch? In my local environment, the test passes.